client-app/src/api/contact/AddressAPI.js

```diff
@@ -8,29 +8,15 @@ export default {
         const AUTH_TOKEN = 'Bearer ' + localStorage.getItem('access_token');
         axios.defaults.headers.common['Authorization'] = AUTH_TOKEN;
 
-        return axios
-            .post(requestUrl, address)
-            .then(function(response) {
-                return response.data.data;
-            })
-            .catch(function(error) {
-                return error.response;
-            });
+        return axios.post(requestUrl, address);
     },
 
     updateAddress: address => {
         const requestUrl = `${URL_ADDRESS}/${address.id}`;
         const AUTH_TOKEN = 'Bearer ' + localStorage.getItem('access_token');
         axios.defaults.headers.common['Authorization'] = AUTH_TOKEN;
 
-        return axios
-            .post(requestUrl, address)
-            .then(function(response) {
-                return response.data.data;
-            })
-            .catch(function(error) {
-                return error.response;
-            });
+        return axios.post(requestUrl, address);
```

Comment:
Ook hier kun je AxiosInstance gebruiken

Reply:
Cleaned up. In AddressAPI alle 4 de functies nu met AxiosInstance.